draft-ietf-scitt-architecture.md

```diff
@@ -339,9 +339,9 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
               v                   '------+----'     |
          .----+----.                     |          |
         |  Signed   |    COSE Signing    |          |
-        | Statement +<-------------------+          |
-         '----+----'                     |          |
-              |                 +--------+------+   |
+        | Statement +<------------------'           |
```

Comment:
The other suggestion is to simply add directionality at the TS, while maintaining the curve to the signed statement
```suggestion
        | Statement +<------------------'|          |
```
![image](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/assets/7647382/b973377e-6ede-409d-b744-1c6e70b62ea2)

Which also needs the suggestion in line 343